waterdrop-core/src/main/scala/io/github/interestinglab/waterdrop/input/sparkstreaming/KafkaStream.scala

```diff
@@ -93,7 +90,10 @@ class KafkaStream extends BaseStreamingInput[(String, String)] {
     }
   }
 
-  override def afterOutput {
+  override def afterOutput(rdd: RDD[Row]) {
+
+    val offsetRanges = rdd.asInstanceOf[HasOffsetRanges].offsetRanges
```

Comment:
java.lang.ClassCastException: org.apache.spark.rdd.MapPartitionsRDD cannot be cast to org.apache.spark.streaming.kafka010.HasOffsetRanges